### ISSUE TITLE
Windows 8 style random tile colors, and new "star"

### ISSUE THREAD

**Issue by cubetronic** [vanilla-music/vanilla#300]
hi, i love the vanilla player. i've been using it for years. i updated a few days ago, and i noticed some changes that i really hate:

when browsing the library by artist, there are now tiles next to artist names. they are super annoying. i absolutely hate the Windows 8 style here. i hate the Windows 8 tiles - it's extremely hard to change the colors - so hard i gave up a long time ago. now it's in vanilla? no! personally, i think "vanilla" should be a pure, clean player, without distractions and extra junk features. i don't mind development that could make things more seamless or whatever, i don't know, but it does everything i need it to already.

also, i absolutely hate the "star" icon... for "favorite-ing" songs? i hate that. i hate that in other players, and i hate to see it here. judging music... if it's in my library, it's because i want it there. if i don't like it, i can delete it. i respect that some people might want to rate their songs and make all sorts of complicated playlists, but in my opinion this is complete b.s.

i couldn't downgrade easily, but thankfully i was able to uninstall and then install 1.0.35, and i don't have these horrible "features" anymore... i never tried version 1.0.36 - i noticed the new stuff in 1.0.37, and i just went back to the oldest available version (without going back to super old versions).

vanilla 1.0.35 is by far the best music player for android.

in summary, from the users perspective: tile colors: they're random! should Rage Against The Machine be lavender? i don't think so. when i play my favorite song, the star is empty. so that means it isn't my favorite? should i tap the star because it *is* my favorite song? what if i don't have *favorite* songs? now i have to organize star/nonstar for all my music? this is madness. and i *will* tap the button if it exists. i always mis-tap everything. it is seriously hard enough to get artist names and song names correct, let alone album art... spare me the random tile colors and America's Got Talent favorite song system.

sorry to be so harsh... i just want it to be Vanilla (not rainbow sherbet), thank you.

**Comment by aminb** [comment]
We could probably have an option for disabling cover arts, but **do keep in mind** that if you're unhappy with Vanilla, no one is forcing you to use it; neither Vanila in its entirety, nor the versions built on F-Droid or for Google Play.

Setting up a build environment and building Vanilla is easy enough, and Vanilla is Free Software. Go ahead and clone the source, revert the commits of features you don't like and build your own version.

**Comment by mindforger** [comment]
`also, i absolutely hate the "star" icon... for "favorite-ing" songs? i hate that. i hate that in other players, and i hate to see it here` 
and i hate people who do not get the use of a feature at all! 

it is not for judging, it's for making a quick playlist ... or have you seen any "1 to 5 stars" rating in this function at all?

seriously calm down and stop whining or read the documentation or the issue that was posted to implement that feature ... https://github.com/vanilla-music/vanilla/issues/277

**Comment by adrian-bl** [comment]
Closing this bug as it is not really a bug.
As @aminb already explained: Vanilla Music is free Software and you are free to use your own fork or you could just stick with an old version.

**Comment by cubetronic** [comment]
The main issue was with the Tile Colors, which no one addressed. I know it's not a bug. It's a feature. But I very much dislike that feature.

> Setting up a build environment and building Vanilla is easy enough, and Vanilla is Free Software. So, you can go ahead, clone the source, revert the commits of features you don't like and build your own version.

All I have is a Chromebook, unfortunately. I tried for a few weeks to get an Android development environment set up in a crouton chroot, but it was a nightmare.

I would be happy with (I think) the 1.0.35 version, or whatever was the last version before the tile colors started, but I can't make the .apk package myself. It would be helpful if that were online.
